### PR TITLE
Dramos/merge branches

### PR DESCRIPTION
**Pull request by daliaramos**
I merged the branches, deleted the node_modules, and added gitignore file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Back-End-Ecomapper-Capstone-Project
 This is the repository for the backend of the Ecomapper capstone project.
+
+Authors:
+Ian G (guyian@pdx.edu)
```
Comment (jgmccarthy7) [inline README.md:5]: All the back end devs should be here, instead of just Ian.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'pug');
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', indexRouter);
```
Comment (jgmccarthy7) [inline app.js:22]: Did you work with Landstrom to port over the work he did onto Ian's index.js? It looks like both app.js and index.js are two separate Node apps as opposed to one app with all the pages.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const express = require("express");
```
Comment (jgmccarthy7) [inline index.js:1]: I think this is a separate app from app.js - which should stay?

**Review by jgmccarthy7 — CHANGES_REQUESTED** [review]
It looks like the two branches got put together and both apps are still separated. The desired behavior is that we have one app that can direct to the different pages you all worked on. Please work with Landstrom to port over the page he made and remove any unneeded code.

**Comment by jgmccarthy7** [comment]
While I am happy to look at pull requests, it's better if you ask for the other members of your team for review of the code - Dan and I will not be as knowledgeable on the code and behavior as your team.

**Review by jgmccarthy7 — APPROVED** [review]
Looks good to me!

**Review by jac0bml — APPROVED** [review]
Addressed in Zoom call with @jgmccarthy7 @daliaramos

**Pull request merged**